optuna_dashboard/ts/components/GraphHyperparameterImportances.tsx

```diff
@@ -64,100 +72,3 @@ const GraphHyperparameterImportanceBackend: FC<{
 
   return <Box component="div" id={plotDomId} sx={{ height: graphHeight }} />
 }
-
-const GraphHyperparameterImportanceFrontend: FC<{
-  studyId: number
-  study: StudyDetail | null
-  graphHeight: string
-}> = ({ studyId, study = null, graphHeight }) => {
-  const theme = useTheme()
-  const colorTheme = usePlotlyColorTheme(theme.palette.mode)
-
-  const numCompletedTrials =
-    study?.trials.filter((t) => t.state === "Complete").length || 0
-  const { importances } = useParamImportance({
-    numCompletedTrials,
-    studyId,
-  })
-  const nObjectives = useStudyDirections(studyId)?.length
-  const objectiveNames: string[] =
-    study?.objective_names ||
-    study?.directions.map((d, i) => `Objective ${i}`) ||
-    []
-
-  useEffect(() => {
-    if (importances !== undefined && nObjectives === importances.length) {
-      plotParamImportance(importances, objectiveNames, colorTheme)
-    }
-  }, [nObjectives, importances, colorTheme])
-
-  return (
-    <Card>
-      <CardContent>
-        <Typography
-          variant="h6"
-          sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
-        >
-          Hyperparameter Importance
-        </Typography>
-        <Box component="div" id={plotDomId} sx={{ height: graphHeight }} />
-      </CardContent>
-    </Card>
-  )
-}
-
-const plotParamImportance = (
-  importances: ParamImportance[][],
-  objectiveNames: string[],
-  colorTheme: Partial<Plotly.Template>
-) => {
-  const layout: Partial<plotly.Layout> = {
-    xaxis: {
-      title: "Hyperparameter Importance",
-    },
-    yaxis: {
-      title: "Hyperparameter",
-      automargin: true,
-    },
-    margin: {
-      l: 50,
-      t: 0,
-      r: 50,
-      b: 50,
-    },
-    barmode: "group",
-    bargap: 0.15,
-    bargroupgap: 0.1,
-    uirevision: "true",
-    template: colorTheme,
-    legend: {
-      x: 1.0,
-      y: 0.95,
-    },
```

Comment:
Could you add this property to `tslib/react/src/components/PlotImportance.tsx`?

Reply:
Added.